infra/helper.py

```diff
@@ -66,6 +66,23 @@
 # TODO(jonathanmetzman): Enforce this.
 DEFAULT_RELATIVE_BUILD_INTEGRATION_PATH = '.cifuzz'
 
+DEFAULT_LANGUAGE = 'c++'
```

Comment:
Can we move these to infra/constants.py ? it is weird to put these in helper.py.